composer.json

```diff
@@ -36,7 +36,7 @@
         "sonata-project/notification-bundle": "^3.3",
         "sonata-project/seo-bundle": "^2.4",
         "sonata-project/twig-extensions": "^0.1 || ^1.2",
-        "sonata-project/user-bundle": "^4.0",
+        "sonata-project/user-bundle": "^4.9",
```

Comment:
Is this needed in this PR?

Reply:
Yes, user-bundle < 4.9 require datagrid 2.x.